int-swap-dApp/src/contracthistory.html

```diff
@@ -66,6 +67,10 @@ <h2>Contract History</h2>
               </table>
               <br/>
                <p> TBD = To be determined </p>
+               <p> Pending = Pending Counterparty or Minting </p>
+               <p> Outstanding = Not Matured</p>
+               <p> Ready to pay = Matured and Payout not Withdraw</p>
```

Comment:
line 71: suggested alternate wording:
Ready to pay = Matured and Payout not Withdrawn Yet